src/molecules/form-fields/text-area-form-field.test.tsx

```diff
@@ -16,4 +19,95 @@ describe("TextAreaFormField", () => {
         // Assert
         expect(getByLabelText(expected)).not.toBeNull();
     });
+
+    test("when has errorMessages prop, renders text area form field with error message", () => {
```

Comment:
suggestion: feel you can remove "text area form field" from the test being it can be inferred from the test fixture